process.go

```diff
@@ -37,18 +37,19 @@ func process(fs fileSystem, filePaths []string) (poFile, error) {
 		if !isSupportedExt(filepath.Ext(filePath)) {
 			continue
 		}
-
-		content, err := fs.readFile(filePath)
-		if err != nil {
-			return poFile, err
+		f, e := os.Open(filePath)
```

Comment:
In order to keep the tests working, this `os.Open` call needs to go through the `fileSystem` interface and then be mocked.

Or better yet, these days it's probably better to rely on `fs.FS`.